composer.json

```diff
@@ -76,7 +77,11 @@
 			"PKP\\plugins\\": "plugins/",
 			"APP\\plugins\\": "../../plugins/",
 			"PKP\\jobs\\": "jobs/",
-			"APP\\jobs\\": "../../jobs/",
+			"APP\\jobs\\": "../../jobs/"
+		}
+	},
+	"autoload-dev": {
+		"psr-4": {
```

Comment:
The "tests" namespaces was introduced by me and aren't used within the application, so I moved them to the `autoload-dev`.